src/main/java/pl/karolska/homework/Age.java

```diff
@@ -0,0 +1,21 @@
+package pl.karolska.homework;
+
+public class Age {
+
+    public static String ageVerification(int age1) {
```

Comment:
Jeżeli metoda zwraca 2 możliwe wartości (w tym przypadku 'jest pełnoletni/ nie jest") to o wiele lepiej pasowałby tutaj inny typ danych zamiast Stringa